socketserver.h

```diff
@@ -456,13 +459,12 @@ class SocketServer
                     {
                         std::cerr << e.what() << '\n';
                     }
-                    
                 }
             } while (true);
 
             close(new_socket);
             ResetReadBuffer();
-            sleep(1);
+            std::this_thread::sleep_for(std::chrono::milliseconds(kSocketRetryDelay));

```

Comment:
I don't think this cast is necessary any more, considering `kSocketRetryDelay` is now declared as an `std::chrono::millisecond` variable?